pkg/metrics/grafana/tidb_resource_control.json

```diff
@@ -225,7 +225,15 @@
               "editorMode": "code",
               "expr": "sum(resource_manager_resource_unit_read_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
               "hide": false,
-              "legendFormat": "{{resource_group}}",
+              "legendFormat": "{{resource_group}}-read",
+              "range": true,
+              "refId": "C"
```

Comment:
A different `refID` should be used here.

Reply:
There seem to be only two in the this panel, I changed it to A and B.